.ci/gcb-generate-diffs.yml

```diff
@@ -153,7 +153,7 @@ steps:
       env:
         - BUILD_ID=$BUILD_ID
         - PROJECT_ID=$PROJECT_ID
-        - BUILD_STEP="19"
+        - BUILD_STEP="19" # also includes steps 20 and 21
```

Comment:
We don't need to adjust these build steps!

The build step serves as a direct link to the cloud build step link. ie: it lets us hop to where we need to be when we click the details link in github!

```
https://console.cloud.google.com/cloud-build/builds;region=global/${build_id};step=${build_step}?project=${project_id}
```

thus if we don't add a new build step won't increment the step counter.. The link will be the same as the old one and thats okay!